### ISSUE TITLE
ColoredImageEffect no longer working in latest versions

### ISSUE THREAD

**Issue by NightScythe1** [elBukkit/MagicPlugin#1230]
I was trying to get a particle image display working, but nothing I tried was working. I copy & pasted the default 'kill' spell instead to see what I could be missing, and found that it doesn't work either (the skull particle image no longer shows)

**Comment by I0nTech** [comment]
> I was trying to get a particle image display working, but nothing I tried was working. I copy & pasted the default 'kill' spell instead to see what I could be missing, and found that it doesn't work either (the skull particle image no longer shows)

Yeah, current attempts at using it just crashed, though imageeffect still works, think this should go to effectlib issues though.
